pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -52,6 +52,8 @@ const (
 	prometheusModuleName   = "prometheus"
 	crashModuleName        = "crash"
 	pgautoscalerModuleName = "pg_autoscaler"
+	balancerModuleName     = "balancer"
+	balancerModuleMode     = "upmap"
```

Comment:
Does the balancer only update when the OSDs are offline? At a glance that's how i'm interpreting [this doc](https://docs.ceph.com/docs/master/rados/operations/upmap/#offline-optimization)

Reply:
no, that part of the doc should be clarified - it describes simulating what the balancer does manually. The balancer effectively automates that process, moving a few pgs every 60s by default, with some throttling to prevent too much movement: https://docs.ceph.com/docs/master/rados/operations/balancer/#throttling